internal/database/release.go

```diff
@@ -771,3 +909,132 @@ func (repo *ReleaseRepo) UpdateBaseURL(ctx context.Context, indexer string, oldB
 
 	return nil
 }
+
+func (repo *ReleaseRepo) CheckIsDuplicateRelease(ctx context.Context, profile *domain.DuplicateReleaseProfile, release *domain.ReleaseNormalized) (bool, error) {
+	queryBuilder := repo.db.squirrel.
+		Select("r.id, r.torrent_name, r.title, ras.action, ras.status").
+		From("release r").
+		LeftJoin("release_action_status ras ON r.id = ras.release_id").
+		Where("ras.status = 'PUSH_APPROVED'")
+
+	if profile.Title {
+		queryBuilder = queryBuilder.Where(sq.Eq{"LOWER(r.title)": release.Title})
+	}
+
+	if profile.ReleaseName {
+		queryBuilder = queryBuilder.Where(sq.Eq{"LOWER(r.torrent_name)": release.TorrentName})
+	}
+
+	if profile.Year {
+		queryBuilder = queryBuilder.Where(sq.Eq{"r.year": release.Year})
+	}
+
+	if profile.Month {
+		queryBuilder = queryBuilder.Where(sq.Eq{"r.month": release.Month})
+	}
+
+	if profile.Day {
+		queryBuilder = queryBuilder.Where(sq.Eq{"r.day": release.Day})
+	}
+
+	if profile.Source {
+		queryBuilder = queryBuilder.Where(sq.Eq{"LOWER(r.source)": release.Source})
+	}
+
+	if profile.Container {
+		queryBuilder = queryBuilder.Where(sq.Eq{"LOWER(r.container)": release.Container})
+	}
+
+	if profile.Codec {
+		var and sq.And
+		for _, codec := range release.Codec {
+			and = append(and, repo.db.ILike("r.codec", "%"+codec+"%"))
+		}
+		queryBuilder = queryBuilder.Where(and)
+	}
+
+	if profile.Resolution {
+		queryBuilder = queryBuilder.Where(sq.Eq{"LOWER(r.resolution)": release.Resolution})
+	}
+
+	if profile.HDR {
+		var and sq.And
+		for _, hdr := range release.HDR {
+			and = append(and, repo.db.ILike("r.hdr", "%"+hdr+"%"))
+		}
+		queryBuilder = queryBuilder.Where(and)
+	}
+
+	if profile.Audio {
+		queryBuilder = queryBuilder.Where(sq.Eq{"r.audio": release.Audio})
+	}
+
+	if profile.Group {
+		queryBuilder = queryBuilder.Where(sq.Eq{"LOWER(r.release_group)": release.Group})
+	}
+
+	if profile.Season {
+		queryBuilder = queryBuilder.Where(sq.Eq{"r.season": release.Season})
+	}
+
+	if profile.Episode {
+		queryBuilder = queryBuilder.Where(sq.Eq{"r.episode": release.Episode})
+	}
+
+	if profile.Website {
+		queryBuilder = queryBuilder.Where(sq.Eq{"r.website": release.Website})
+	}
+
+	if release.Proper {
```

Comment:
Do you mean why is `release.Proper` not parallel with the other cases?  I tried to explain my reasoning in commit 915094013eb3af53b05873db8fcf035ba146987e.  Proper is not at all like the other profile conditions, and I think it should be removed from Duplicate Profiles altogether, but this condition should remain so that a PROPER is never considered a duplicate.

Reply:
Resolving because I answered the feedback.  Feel free to reopen if I didn't address it sufficiently.

Reply:
That's unfortunate.  So that sounds to me like a vote to remove Proper and Repack from Duplicate Profiles, and also to remove the Proper/Repack checks from `CheckIsDuplicateRelease`.  Did I read you right @KyleSanderson?  Then at least we can skip all duplicates, and the downside is that we won't *automatically* get REPACK/PROPER fixes.